src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -75,6 +75,9 @@ target_include_directories(${TARGET_NAME} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/sr
 
 target_include_directories(${TARGET_NAME} SYSTEM PRIVATE
     $<TARGET_PROPERTY:dnnl,INCLUDE_DIRECTORIES>)
+if (CMAKE_BUILD_TYPE STREQUAL Debug)
```

Comment:
how it works for multi-config generators? E.g. MSVC which provides selection between debug / release after I configured cmake project. I suppose we need to use generator expressions here (see https://stackoverflow.com/questions/34490294/what-does-configdebugrelease-mean-in-cmake)

Reply:
Done